pkg/lib/files/files.go

```diff
@@ -552,13 +555,44 @@ func PromptForFilesAboveSize(size int, promptMsgTemplate string) IgnoreFn {
 
 	return func(path string, fi os.FileInfo) (bool, error) {
 		if !fi.IsDir() && fi.Size() > int64(size) {
-			promptMsg := fmt.Sprintf(promptMsgTemplate, PathRelativeToCWD(path), s.IntToBase2Byte(int(fi.Size())))
+			promptMsg := fmt.Sprintf(promptMsgTemplate, PathRelativeToCWD(path), s.Int64ToBase2Byte(fi.Size()))
 			return !prompt.YesOrNo(promptMsg, "", ""), nil
 		}
 		return false, nil
 	}
 }
 
+func ErrorOnBigFilesFn(maxFileSizeBytes int64, maxMemoryUsagePercent float64) IgnoreFn {
+	return func(path string, fi os.FileInfo) (bool, error) {
+		if !fi.IsDir() {
+			fileSizeBytes := fi.Size()
+			virtual, _ := mem.VirtualMemory()
+			if float64(fileSizeBytes) > float64(virtual.Available) ||
```

Comment:
`float64(fileSizeBytes) > float64(virtual.Available) ||` condition may not be necessary